test/build.spec.ts

```
@@ -121,20 +117,26 @@ describe('module builder', () => {
+        injection: "injected";
+      }>;
+      //#endregion
+      export { SharedTypeFromRuntime, _default as default };"
+    `)
   })
 
   it('should correctly add extensions to imports from runtime/ directory', async () => {
     const moduleDts = await readFile(join(distDir, 'module.d.mts'), 'utf-8')
     const runtimeImport = findStaticImports(moduleDts).find(i => i.specifier.includes('runtime'))
-    expect(runtimeImport!.code.trim()).toMatchInlineSnapshot(`"import { SharedTypeFromRuntime } from '../dist/runtime/plugins/plugin.js';"`)
+    expect(runtimeImport!.code.trim()).toMatchInlineSnapshot(`"import { SharedTypeFromRuntime } from \"./runtime/plugins/plugin.js\";"`)
```

ESLint

[error] Unnecessary escape character: \".

(no-useless-escape)